src/devtools/client/webconsole/components/Output/MessageIcon.js

```diff
@@ -27,12 +27,9 @@ function getIconElement(level, onRewindClick, type) {
   let title = l10n.getStr(l10nLevels[level] || level);
   const classnames = ["icon"];
 
-  if (type === "logPoint") {
-    classnames.push("logpoint");
```

Comment:
Is there a CSS class that needs removed as well?

Reply:
yep! fixed